### DISCUSSION TITLE
[Question]: Is it possible to configure both OpenAI and Azure OpenAI assistants?

### DISCUSSION THREAD

**Discussion by dividor** [danny-avila/LibreChat#2440] [Q&A] [answered]
### What is your question?

I would like to use LibreChat for comparing the performance of OpenAI assistants vs. Azure OpenAI assistants. I can get one or the other to work, but following the docs I couldn't seem to get both working in the same session.

Is having assistants from both platforms at the same time currently supported please?

### More Details

.

### What is the main subject of your question?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
No, at the moment, you can't use azure and non-azure at once.

**Reply by danny-avila** [reply]
In fact, with azure, it is tricky even using more than one resource group if they don't share region. There are a couple of reasons why supporting both at the same time isn't possible, primarily with Assistants being developed before there was even Azure support.

**Reply by danny-avila** [reply]
This is also not out of the question to develop, but isn't a priority at the moment. I've created milestones to better illustrate upcoming progress: https://github.com/danny-avila/LibreChat/milestones

**Comment by danny-avila** [comment]
@dividor just FYI due to the v2 migration, I'm going to be making azure its own endpoint.
